client/src/components/ScoreForm.js

```diff
@@ -80,11 +137,22 @@ const TableBonuses = ({ bonuses, handleCheckedBox }) => (
 	</Table>
 )
 
-const CurrIntegerExercise = ({ value, units }) => (<div>{value} {units}</div>)
+const Bonus = field => (
+	<Checkbox
+		{...field.input}
+		value={field.input.value ? 'on' : 'off'}
+		onChange={(e, { checked }) => field.input.onChange(checked)}
+		label={field.label} 		
+	/>
+)
+
+const CurrIntegerExercise = ({ currValue, units }) => <div>{currValue} {units}</div>
```

Comment:
Is this setup this way because our sample store has the data in hours and minutes? Will this need to do a time conversion from minutes or will something else do that?

Reply:
Our sample store has data in hours and minutes because this needs to be set up like this as redux form needs to have separate components for hours and minutes to function correctly. We can do a time conversion from minutes between receiving the data from the server and populating the store.